media/css/firefox/firstrun/firstrun_quantum.less

```diff
@@ -0,0 +1,542 @@
+@import "../../sandstone/lib.less";
+
+@font-face {
+    font-family: 'Fira Sans Light';
+    src: url('/media/fonts/FiraSans-Light.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-Light.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Fira Sans Light';
+    src: url('/media/fonts/FiraSans-SemiBold.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-SemiBold.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+}
+
+:root {
+  --animation-curve: linear;
+  // --animation-curve: cubic-bezier(.73,.08,.94,.33);
+  // --backwards: cubic-bezier(.49,-0.41,.96,.67);
+}
+
+body {
+    margin: 0;
+    overflow: hidden;
```

Comment:
This is preventing any scrolling, but scrolling is probably essential for smaller windows. You might not be able to scroll down to the button, for example. We should probably build in some responsiveness (see other comments elsewhere) and at least allow for some vertical scrolling when needed. I'm not sure how to make that work with the wavy background though since that pretty much has to overflow. Maybe it could be in a wrapper sized to the viewport with `overflow: hidden` and `position: fixed;`, then the content on a separate layer on top of it? Worth a try maybe?

Reply:
I've set each image to `position: fixed`, and changed the overflow-y on a height of 710px.